src/con.c

```diff
@@ -1631,6 +1631,13 @@ Con *con_descend_direction(Con *con, direction_t direction) {
     return con_descend_direction(most, direction);
 }
 
+bool has_outer_gaps(gaps_t gaps) {
```

Comment:
Please make this function `static`, otherwise Travis will complain with the warnings enabled there.